entgql/template/pagination.tmpl

```diff
@@ -631,7 +631,10 @@ func ({{ $r }} *{{ $name }}) ToEdge(order *{{ $order }}) *{{ $edge }} {
 	if hasCollectedField(ctx, totalCountField) || hasCollectedField(ctx, pageInfoField) {
 		hasPagination := after != nil || first != nil || before != nil || last != nil
 		if hasPagination || ignoredEdges {
-			if conn.TotalCount, err = {{ $r }}.Clone().Count(ctx); err != nil {
+			c := {{ $r }}.Clone()
+			// We don't need to select the fields of the node, since we only need the count.
```

Comment:
Change it to a template comment instead? There’s no need having this in all generated code.